website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -58,7 +58,9 @@ Notifications are sent as the run progresses, and can be triggered on one or mor
 | Errored            | `"run:errored"`         | A run terminates early due to error or cancellation.                                          |
 | Drifted            | `"assessment:drifted"`  | Terraform Cloud detected configuration drift. This option is only available if you enabled drift detection for the workspace.                                                    |
 | Checks Failed      | `"assessment:check_failure"` | One or more continuous validation checks did not pass. This option is only available if you enabled drift detection for the workspace. |
-| Health Assessment Failed | `"assessment:failed`"   | A health assessment failed. This option is only available if you enable health assessments for the workspace.                                        |
+| Health Assessment Errored | `"assessment:failed`"   | A health assessment failed. This option is only available if you enable health assessments for the workspace.                                        |
+| Auto Destroy Reminder | `"workspace:auto_destroy_reminder`"   | Reminders that a destroy run will be triggered at some point in the future. Reminders are sent 12 and 24 hours before the scheduled auto destroy time. |
+| Auto Destroy Results | `"workspace:auto_destroy_run_results`"   | Final run result for the scheduled auto destroy run. |
```

Comment:
💅 Nit: A subtle note on how these docs differ structurally from the rest. More or less all the other descriptions start by describing the trigger for the notification. These might opt to start with the following, respectively:
`An automated workspace destroy run is imminent.`
`An automated destroy run was attempted.`

Reply:
great point